scripts/setSignerVersion.mts

```diff
@@ -12,15 +13,15 @@ const { values: { [VERSION_PARAM_NAME]: version = '' }} = parseArgs({
   }
 })
 
-assert(/\d\.\d\.\d/.test(version), `The "${VERSION_PARAM_NAME}" argument: "${version}" does not match the "x.x.x" format.`)
+assert(/^\d+\.\d+\.\d+(-alpha.\d+)?$/.test(version), `The "${VERSION_PARAM_NAME}" argument: "${version}" does not match the "x.x.x" format.`)
 
-const replacementResult = await replace.replaceInFile({
-  files: 'packages/extension/build/**',
-  from: /{{signer-version}}/g,
-  to: version,
-  countMatches: true,
-})
+const currentPackageJsonContent = await readFile(EXTENSION_PACKAGE_JSON_PATH, 'utf8')
```

Comment:
doesn't really matter, but we could use `readFileSync` here :shrug:

Reply:
I prefer to use the async counterparts by default - it's safer from maintainability perspective (e.g. any kind of future refactors won't have the burden of blocked synchronicity) and almost always slightly more performant (for the same reason - freeing the thread) for very little cost.